exercises/D-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return 10 > number > 0;
+}
```

Comment:
Seems like your function only checks if number less than 10.
Have you tried testing this function with different number for example 0, -1, 1, 11? 
Try this way, it should work - `10 > number && number > 0`